packages/release-trigger/src/index.d.ts

```diff
@@ -0,0 +1,18 @@
+// Copyright 2021 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+declare module 'promise-events' {
```

Comment:
I've seen this a few times throughout the repo, did some digging, and stumbled across [this thread](https://github.com/googleapis/repo-automation-bots/pull/44/files#r316901680) from 2019. Do we still need this file?